lib/Search/Provider.php

```diff
@@ -38,9 +43,11 @@ function search($query) {
 		$userId = \OCP\User::getUser();
 		$ownerView = new \OC\Files\View('/'. $userId . '/files');
 
+		$model = intval($this->config->getAppValue('facerecognition', 'model', AddDefaultFaceModel::DEFAULT_FACE_MODEL_ID));
+
 		$searchresults = array();
 
-		$results = $this->faceMapper->findAllNamedLike($userId, $query);
```

Comment:
if this is last usage of findAllNamedLike, consider removing it completely (as there is no logic for accessing it anymore)